src/tracing/js/bindings.rs

```diff
@@ -727,6 +727,9 @@ impl JsEvmContext {
         if let Some(tx_hash) = transaction_ctx.tx_hash {
             obj.set(js_string!("txHash"), to_byte_array(tx_hash.0, ctx)?, false, ctx)?;
         }
+        if let Some(coinbase) = transaction_ctx.coinbase {
+            obj.set(js_string!("coinbase"), address_to_byte_array(coinbase, ctx)?, false, ctx)?;
```

Comment:
we need to get this from the `blockenv` instead:

https://github.com/paradigmxyz/revm-inspectors/blob/d55d3c156081980c803abd1c7062829aa9db8db9/src/tracing/js/mod.rs#L263-L263

via `env.blockenv.coinbase`

Reply:
cool, we have `env.block`, so no need to adjust reth's codebase.